tests/python/tests/http_server/test_request.py

```diff
@@ -9,6 +11,25 @@ def test_send_simple_request_path(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.content, b"Hello world!")
 
+    def test_send_request_with_early_hints(self):
+        msg = b"GET /?hints=yes HTTP/1.1\n\n"
+        with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+            s.connect(('127.0.0.1', self.kphp_server.http_port))
+            s.send(msg)
+            early_hints = s.recv(len(msg))
```

Comment:
let's settimeout(0.1) before that to ensure early hints is received right after call of `send_http_103_early_hints` on server side